src/Parsers/ParserCreateQuery.h

```diff
@@ -105,9 +105,9 @@ class IParserColumnDeclaration : public IParserBase
 
     bool parseImpl(Pos & pos, ASTPtr & node, Expected & expected) override;
 
-    bool require_type = true;
-    bool allow_null_modifiers = false;
-    bool check_keywords_after_name = false;
+    const bool require_type = true;
```

Comment:
Default values for bools are unneeded as all members are initialized in the ctor in l. 93ff.

Reply:
I consider that it's better to always initialize to avoid uninitialized members in case the constructor will change or new added